### PR TITLE
adds support for GCC 11

### PR DESCRIPTION
**Pull request by fpelliccioni**
Changelog: (Feature | Fix | Bugfix): Describe here your pull request

- [ ] Refer to the issue that supports this Pull Request.
- [x] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [x] I've read the [Contributing guide](https://github.com/conan-io/conan-docker-tools/blob/master/.github/CONTRIBUTING.md).
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008) style guides for Python code.
- [x] I've followed the [Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) guides for Dockerfile.

### CODE REVIEW

File: gcc_11/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+       sudo \
+       binutils \
+       wget \
+       git \
+       libc6-dev \
+       g++-${GCC_VERSION} \
```
Comment (Croydon) [inline gcc_11/Dockerfile:21]: This image was not build as you have not added it to the `docker-compose.yml` file.

`gcc-11` does not exist for Ubuntu Focal

**Review by Croydon — CHANGES_REQUESTED** [review]

**Comment by uilianries** [comment]
Please, consider https://github.com/fpelliccioni/conan-docker-tools/pull/1

This PR is outdated and Conan target version need to be updated.

**Comment by fpelliccioni** [comment]
> Please, consider [fpelliccioni#1](https://github.com/fpelliccioni/conan-docker-tools/pull/1)
> 
> This PR is outdated and Conan target version need to be updated.

merged, thanks!

**Comment by uilianries** [comment]
Thank you!

**Pull request merged**